tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -59,6 +59,15 @@ def _construct_reconfiguraiton_wedge_coammand(self):
         reconf_msg.additional_data = bytes()
         return reconf_msg
 
+    def _construct_reconfiguraiton_latest_prunebale_block_coammand(self):
```

Comment:
Nit: change `_construct_reconfiguraiton_latest_prunebale_block_coammand` to `_construct_reconfiguration_latest_pruneable_block_command`.

Reply:
done